### ISSUE TITLE
Create page “Is the data open that matters to me?” 

### ISSUE THREAD

**Issue by dannylammerhirt** [okfn/opendatasurvey#962]
GODI emphasises that we assess sometimes rather general, sometimes quite specific data. Currently we  communicate this in much detail to our reviewers, but not to other audiences. This is a missed opportunity

- [ ] governments need to understand which data we base our ranking on
- [ ] civic interest groups should learn what data we assess, and why it matters to have this data available as open data

Action points: 
- [ ] Create a stand-alone page for that, named “Is your relevant data open?”
- [ ] Add link to this “Is data open that matters to me?” page on homepage.
- [ ] Add data category descriptions on the page above
- [ ]Add use cases on the same page

**Comment by dannylammerhirt** [comment]
Content available in this working document
https://docs.google.com/document/d/1h_a4r1SO58T0m31AAWz4ioPTRr7PrIomoc9a8Pev3B4/edit

**Comment by morchickit** [comment]
@dannylammerhirt - I think we can close it, no?

**Comment by dannylammerhirt** [comment]
Yes, it's contained in other blogposts like "How to read the GODI results"
